pkg/booleanpolicy/augmentedobjs/construct.go

```diff
@@ -236,43 +236,22 @@ func ConstructImage(image *storage.Image) (*pathutil.AugmentedObj, error) {
 	}
 
 	if features.ImageSignatureVerification.Enabled() {
-		i := 0
-		// Since policies query for image verification status as a single boolean field, we add it to the image here.
+		ids := []string{}
```

Comment:
pre-allocate:
`id := make([]string, 0, len(image.GetSignatureVerificationData().GetResults()))`

Reply:
So we consider it alright to let the slice be written multiple times as it expands?

Reply:
Why would that be a problem? Multiple times = a few, most probably once.